src/api/index.js

```diff
@@ -23,17 +23,16 @@ const config = new Conf();
   * Lists plugins & themes with the keyword 'dext-plugin' or 'dext-theme' on npm
   *
   * @param {String} plugin - The name of the plugin/theme
-  * @return {Promise} - Resolves the search results as an array
+  * @return {Promise} - Resolves the search results
   */
 const search = searchTerm => new Promise((resolve, reject) => {
   searchPackages(searchTerm).then((packages) => {
-    if (Array.isArray(packages)) {
-      // Loop over all found packages to return a list
-      const results = packages.map(pkg => pkg.name[0]);
-      resolve(results);
-    } else {
+    if (!Array.isArray(packages) || !packages.length) {
```

Comment:
If no packages are found for the search it will result in a reject (and an error in the cli) (because of !packages.length) and maybe you just want to reject if the search actually fails.